### DISCUSSION TITLE
side-by-side view; multiple windows

### DISCUSSION THREAD

**Discussion by skaunov** [Zettlr/Zettlr#4483] [Q&A] [answered]
Hi, I feel like I'm the only Debian user who wanted to use Zettlr in couple of windows side-by-side: can't find this feature, its description, or discussion. _Could you give me a relevant link, or create a feature request if there's none, or explain to me why is it a bad idea._

(I use Zettlr Deb package with Pop OS.)

**Comment by exanrauzer** [comment]
Which version are you using?

**Comment by skaunov** [comment]
Didn't thought about it... Should be latest `deb` package from the site. Will check when get to the laptop.On Jul 7, 2023 4:59 PM, Exan Rauzer ***@***.***> wrote:
Which version are you using?

**Reply by exanrauzer** [reply, marked as answer]
Go to https://github.com/Zettlr/Zettlr/releases and grab beta5. You gonna got all you need.

**Reply by skaunov** [reply]
I gain ability to open second window, but lost ability to open the file just double-clicking. I can live with it, thnx!

**Reply by exanrauzer** [reply]
What do you mean? Explain please. I am eager to spot on this.

**Reply by skaunov** [reply]
double-clicking on a Md-file at the desktop opens Zettlr with nothing; before update it opened Zettlr with the chosen file presented
@exanrauzer
